<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_bewegen_hoch">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>410</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Hoch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_links">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>440</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Links</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_runter">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>470</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Runter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_naechste">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>470</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>NEXT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drehen_links">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>410</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Links Drehen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_drehen_rechts">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>410</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rechts Drehen</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_bewegen_rechts">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>440</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Rechts</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_reset">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>470</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RESET</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_flip">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>440</y>
      <width>130</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>FLIP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_logo">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>530</y>
      <width>741</width>
      <height>180</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Chiller</family>
      <pointsize>120</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TI BONGO</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_punkte">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Punkte</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zeit">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Restzeit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_eingaben">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>70</y>
      <width>300</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eingaben</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_punkte">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>10</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_eingaben">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>70</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_zeit">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>110</width>
      <height>60</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein1">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stein 1</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein2">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stein 2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stein 3</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_stein4">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Stein 4</string>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_Spielsteine">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>400</width>
      <height>170</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="graphicsView_Spielfeld">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <zorder>graphicsView_Spielsteine</zorder>
   <zorder>pushButton_bewegen_hoch</zorder>
   <zorder>pushButton_bewegen_links</zorder>
   <zorder>pushButton_bewegen_runter</zorder>
   <zorder>pushButton_naechste</zorder>
   <zorder>pushButton_drehen_links</zorder>
   <zorder>pushButton_drehen_rechts</zorder>
   <zorder>pushButton_bewegen_rechts</zorder>
   <zorder>pushButton_reset</zorder>
   <zorder>pushButton_flip</zorder>
   <zorder>label_logo</zorder>
   <zorder>label_punkte</zorder>
   <zorder>label_zeit</zorder>
   <zorder>label_eingaben</zorder>
   <zorder>lcdNumber_punkte</zorder>
   <zorder>lcdNumber_eingaben</zorder>
   <zorder>lcdNumber_zeit</zorder>
   <zorder>radioButton_stein1</zorder>
   <zorder>radioButton_stein2</zorder>
   <zorder>radioButton_stein3</zorder>
   <zorder>radioButton_stein4</zorder>
   <zorder>graphicsView_Spielfeld</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
